src/istgt_lu_disk.c

```diff
@@ -269,9 +270,24 @@ find_first_bit(ISTGT_LU_CMD_Ptr lu_cmd)
 				:"=a" (res), "=&c" (d0), "=&D" (d1)
 				:"0" (0ULL), "1" (nbits), "2" (addr),
 				[addr] "r" (addr) : "memory");
-		res_final +=res;
-		if((unsigned long)res != nbits)
+		res_final += res;
+		if ((unsigned long)res != nbits)
```

Comment:
can we fix it for this case also as you did for __aarch64__? Can we put a check  `if ((unsigned long)res != nbits << 6)` here also.